internal/server/submit/conversion/conversions_test.go

```diff
@@ -4,6 +4,8 @@ import (
 	"math"
 	"testing"
 
+	"github.com/armadaproject/armada/internal/server/configuration"
```

Comment:
this needs to be in the block with the armada imports at the bottom (annoyingly linting isn't intellligent enough to fix/catch this)

Reply:
this is weird, `goimports` should catch this, I will check